stdlib/public/CMakeLists.txt

```diff
@@ -19,6 +19,19 @@ if(SWIFT_STDLIB_SIL_DEBUGGING)
   list(APPEND SWIFT_RUNTIME_SWIFT_COMPILE_FLAGS "-Xfrontend" "-sil-based-debuginfo")
 endif()
 
+option(SWIFT_RUNTIME_PART_OF_DARWIN_SHARED_CACHE
```

Comment:
Personally, I think that `SWIFT_RUNTIME_DYLD_SHARED_CACHE_COMPATIBLE` is a better option name.

Reply:
Yes, I agree -- it makes the code more readable, since there is a visual match for "dyld shared cache" in the condition and the flag applied.

Applied this suggestion in [d84140d](https://github.com/apple/swift/commit/d84140d1d331b7c665bb8ea72c79be297191444d)